client/ayon_core/tools/publisher/control.py

```diff
@@ -150,10 +150,15 @@ def _create_plugin_data_item(self, plugin):
         label = None
         if hasattr(plugin, "label"):
             label = plugin.label
-
+        result = inspect.getmro(plugin)
+        plugin_type = result[1].__name__
         return {
             "id": plugin.id,
             "name": plugin.__name__,
+            "filepath": inspect.getfile(plugin),
+            "doc": plugin.__doc__,
```

Comment:
Should this maybe still go through [`inspect.cleandoc`](https://docs.python.org/3/library/inspect.html#inspect.cleandoc)?

If so, we might be better of directly going through [`inspect.getdoc(plugin)`](https://docs.python.org/3/library/inspect.html#inspect.getdoc)?

Reply:
I now use 
`inspect.getdoc(plugin)`